add fqn consideration to auto exposures (#6991)
dbt-labs/docs.getdbt.com: website/snippets/_auto-exposures-considerations-tb.md

Auto-exposures with Tableau have the following considerations:
- You can only connect to a single Tableau site on the same server.
- If you're using Tableau Server, you need to [allowlist dbt Cloud's IP addresses](/docs/cloud/about-cloud/access-regions-ip-addresses) for your dbt Cloud region.
- Tableau dashboards built using custom SQL queries aren't supported.
- Auto-exposures sync automatically _once per day_ or when a user updates the selected collections.

> - <Expandable alt_header="The database Fully qualified names (FQNs) in Tableau must match those in the dbt build.">

Reviewer: ```suggestion
- <Expandable alt_header="The database fully qualified names (FQNs) in Tableau must match those in the dbt build.">
```

>     The database FQNs (Fully qualified names) in Tableau must match those in the dbt build. In order to view all expected dependencies in your exposure, the FQNs must match and it isn't case sensitive. For example:

Reviewer: ```suggestion
    Tableau's database FQNs (fully qualified names) must match those in the dbt build. To view all expected dependencies in your exposure, the FQNs must match but aren't case-sensitive. For example:
```

    | Tableau FQN | dbt FQN | <div style={{width:'250px'}}>Result</div> |
    | --- | --- | --- |
    | `analytics.dbt_data_team.my_model` | `analytics.dbt_data_team.my_model` | ✅  Matches and dependencies will display as expected.|
    | `analytics.dbt_data_team.my_model` | `prod_analytics.dbt_data_team.my_model` | ❌ Doesn't match and not all expected dependencies will display. |

    To troubleshoot this:

>     1. In dbt Cloud, download the `manifest.json` from the most recent production run that includes the missing dependencies.

Reviewer: Should we tell them how to do this?

Author: add info on how to access via artifacts tab in job run

            ```jsx
                query {
                  workbooks {
                    name
                    uri
                    id
                    luid
                    projectLuid
                    projectName
                    upstreamTables {
                      id
                      name
                      schema
                      database {
                        name
                        connectionType
                    }
                  }
                }
              }
            ```

    3. Compare database FQNs between `manifest.json` and the GraphiQL response. Make sure that `{database}.{schema}.{name}` matches in both.
        <Lightbox src="/img/docs/cloud-integrations/auto-exposures/manifest-json-example.png" title="manifest.json example"/>
        <Lightbox src="/img/docs/cloud-integrations/auto-exposures/graphiql-example.png" title="GraphiQl response example"/>
    4. If the FQNs don't match, update your Tableau FQNs to match the dbt FQNs.

>     5. If you're still experiencing issues, please reach out to dbt Support and share the results with them.

Reviewer: ```suggestion
    5. If you're still experiencing issues, please contact [dbt Support](mailto:support@getdbt.com) and share the results with them.
```

Author: good callout, thank yoU!
